### DISCUSSION TITLE
Lights flashing instead of turning on

### DISCUSSION THREAD

**Discussion by TheOriginal92** [Koenkk/zigbee2mqtt#22096] [General]
Hello, all of my zigbee2mqtt connected lights have a wired behavior. When I turn them on, they usually do a short (< 0.5 sek) flash and then stay off. 
when i adjust brightness they then turn on and can be controlled normally. This happens with openhab and home assistant. After the lights turned on once they can be controlled normally for some time.

Anyone had a similar experience?

Thank you!

**Comment by lux73** [comment]
I had the same experience with a cheap MiBoxer RGBWW Controller, i throw it out from my Network 🙃

Which Hardware or is this with all of your Lights?

**Reply by TheOriginal92** [reply]
Actually it’s all the lights. Not a specific brand that does not show this behavior.

**Comment by julianfs** [comment]
I have seen this on my network in the past, but not recently. I suspect it was when turning lights on to a low brightness setting, perhaps with a transition, which causes the bulb to be seen as 'off' before reaching a higher brightness (if that makes sense).

What is your zigbee coordinator?

**Reply by TheOriginal92** [reply]
Hmm, that sounds interesting. Infact all my lights use a 0.5 sek transition. Maybe I should try and turn this off. 
Coordinator: Egony Stick V4 with CC2652P Coordinator-Type: zStack3x0
Coordinator-Version: 20230507

**Reply by lux73** [reply]
i doubt that transition is the source of error - all my Lights have transition from 0.3-0.5 Sek

but it's worth a try

**Reply by TheOriginal92** [reply]
I removed the delays, Nothing changed. But I found out that this also happens when the lights are turned on using the z2m web interface. So this does not seem to be Mqtt or HA/OH related

**Comment by yuanxin32323** [comment]
Me too. To solve this problem, currently my "automation" operation does not use "services", but calls the scene function built into zigbee2mqtt. It can alleviate some problems, but the voice control still has the situation of the light flashing and then turning off.
